### PR TITLE
More MC -> M€ conversion.

### PR DESCRIPTION
**Pull request by kberg**

### CODE REVIEW

File: src/styles/turmoil.less [hunk 1]
```diff
@@ -352,7 +352,7 @@
     margin: auto;
 }
 
-.label-5mc {
+.label-5M€ {
```
Comment (alrusdi) [inline src/styles/turmoil.less:355]: Is it intended? css class names with non-latin symbols are not recommended

Comment (kberg) [reply src/styles/turmoil.less:355]: Damn! I thought I checked them all. Sorry.

Comment (kberg) [reply src/styles/turmoil.less]: Also, I think this CSS class is unused.

Comment (kberg) [reply src/styles/turmoil.less]: Should be OK now. Sorry about that. I'm going to make smaller PRs ongoing.

**Review by alrusdi — APPROVED** [review]

**Pull request merged**